data/map.txt

```diff
@@ -5582,6 +6146,78 @@ system "Beta Lupi"
 		distance 1976.61
 		period 1406.05
 
+system "Beta Lyrae"
+	pos -120.44445 65.444447
+	government Syndicate
+	attributes core
+	link "Beta Carinae"
+	link Messiah
+	link Ruchbah
+	link Sheratan
+	asteroids "small rock" 30 5.0232
+	asteroids "medium rock" 34 5.2052
+	asteroids "large rock" 11 1.82
+	asteroids "small metal" 4 4.6228
+	asteroids "medium metal" 199 2.4024
+	asteroids "large metal" 46 4.4044
+	minables iron 61 3.64072
+	haze _menu/haze-133
+	fleet "Small Core Merchants" 800
+	fleet "Large Core Merchants" 1000
+	fleet "Small Syndicate" 1600
+	fleet "Large Syndicate" 2500
+	fleet "Small Core Pirates" 2000
+	fleet "Large Core Pirates" 4000
+	habitable 2560
+	belt 1130
+	trade Clothing 310
+	trade Electronics 750
+	trade Equipment 673
+	trade Food 174
+	trade "Heavy Metals" 860
+	trade Industrial 774
+	trade "Luxury Goods" 1013
+	trade Medical 671
+	trade Metal 292
+	trade Plastic 376
+	object
+		sprite star/o-supergiant
+		distance 371
+		period 357.29823
+	object
+		sprite star/f-supergiant
+		distance 371
+		period 357.29823
+		offset 180
+	object
+		sprite planet/gas10
+		distance 1162
+		period 700.2195
+	object
+		sprite planet/jupiter-b
+		distance 1884
+		period 1445.5941
+	object
+		sprite planet/neptune-b
+		distance 2544
+		period 2268.3009
+	object
+		sprite planet/browndwarf-y-rouge
```

Comment:
This one has got to be the worst.  A shadowless brown dwarf in a system with binary purple and white supergiants?  That's even more insane than the idea of this system.  And in human space no less!

```suggestion
		sprite planet/browndwarf-y
```